src/centralserver/admin-service/src/main/resources/openapi-definition.yaml

```diff
@@ -1,357 +1,4105 @@
+            </li>
+            <li>pin_code_exists</li>
+            <li>pin_code_not_provided</li>
+            <li>server_address_exists</li>
+            <li>server_address_not_provided</li>
+            <li>instance_identifier_exists</li>
+            <li>instance_identifier_not_provided</li>
+            <li><code>validation_failure</code> for parameter format errors
+          </ul>
+          <strong>See the '400' response examples</strong>
+        </p>
+      operationId: initCentralServer
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InitialServerConf'
+        description: initial central server configuration
+      responses:
+        '201':
+          description: central server initialized
+        '400':
+          content:
+            application/json:
+              examples:
+                error_invalid_init_params:
+                  $ref: '#/components/examples/ErrorInvalidInitParamsMetadataExample'
+                error_response:
+                  $ref: '#/components/examples/ErrorExample'
+                error_weak_pin:
+                  $ref: '#/components/examples/ErrorWeakPinMetadataExample'
+                validation_errors_response:
+                  $ref: '#/components/examples/ValidationErrorsExample'
+                warnings_response:
+                  $ref: '#/components/examples/InitializationWarningExample'
+              schema:
+                $ref: '#/components/schemas/ErrorInfo'
+          description: request was invalid
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+      summary: Initialize a new central server with the provided initial configuration
+      tags:
+        - initialization
+  /initialization/status:
+    get:
+      description: <h3>Administrator checks the initialization status of the Central Server.</h3>
+      operationId: getInitializationStatus
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InitializationStatus'
+          description: initialization status of the Central Server
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+      summary: Check the initialization status of the Central Server
+      tags:
+        - initialization
+  /intermediate-cas/{hash}:
```

Comment:
In old app we can have multiple intermediate ca's with same certs = same hashes, so we may need to change handle from hash to pk id. 

Question: https://niis.slack.com/archives/GGBHUKL68/p1643899711584589

Reply:
It seems we don't know for sure that hashes are unique keys. Using opaque identifiers (DB PKs) instead of hashes now. 

Addressed in https://github.com/nordic-institute/X-Road/pull/1164/commits/f252d729132e7cb2b3d1642d452f756e0154b1bc